### DISCUSSION TITLE
Postfixadmin broke after Roundcube install

### DISCUSSION THREAD

**Discussion by phipac** [postfixadmin/postfixadmin#661] [Q&A]
Greetings, friends. I'm having difficulty locating a fix for a particular problem that has arisen. I had access to Postfixadmin working fine. After following a tutorial to install Roundcube, I now can no longer access Postfixadmin with the superadmin account. The following errors are present when I log in with the setup password:

- Password Hashing - attempted to use configured encrypt backend (dovecot:ARGON2I) triggered an error: /usr/bin/doveadm pw -r 5 failed, see error log for details
- You will have problems logging into PostfixAdmin.
- Check out our Dovecot documentation at https://github.com/postfixadmin/postfixadmin/blob/master/DOCUMENTS/DOVECOT.txt, specifically around '3. Permissions'.

Apparently something changed with the encryption method, or else I'm reading it wrong. Syslog doesn't show any dovecot errors. My environment is Ubuntu 22.04. Is there somewhere else I can look for errors in a log? Thank you again for helping me out! I appreciate all your time and efforts.

Phil

**Comment by DavidGoodwin** [comment]
I'd suggest you start here - https://github.com/postfixadmin/postfixadmin/blob/master/DOCUMENTS/DOVECOT.txt#L143

Your problem is that you've upgraded dovecot somehow, and when 'doveadm' is being called, it's not able to read the ssl keys you've configured dovecot with.

The referenced github tickets - e.g. https://github.com/postfixadmin/postfixadmin/issues/381 and https://github.com/postfixadmin/postfixadmin/issues/398 may also help
